src/components/editor/MetapageEditor.tsx

```diff
@@ -8,15 +8,15 @@ export type EditorProps = {
   readOnly?: boolean;
 };
 
-export const Editor: React.FC<EditorProps> = ({
+export const MetapageEditor: React.FC<EditorProps> = ({
```

Comment:
It's a MetaframeEditor, not a MetapageEditor. This site is a metaframe, and not all use cases are in metapages directly.